releasenotes/notes/compress-processes-34bb37ada5d05abb.yaml

```diff
@@ -0,0 +1,3 @@
+enhancements:
+  - |
+    Compress legacy processes metadata payload.
```

Comment:
Note: I do not think a release note is needed for this change as it is more an internal change.